packages/bot-web-ui/src/components/dashboard/dashboard.tsx

```diff
@@ -164,13 +164,20 @@ const Dashboard = ({
         const dbot_settings = JSON.parse(localStorage.getItem('dbot_settings') as string);
         const has_onboard_token_set = active_tab === 0 && !dbot_settings?.onboard_tour_token;
         const has_bot_builder_token_set = active_tab === 1 && !dbot_settings?.bot_builder_token;
+        const show_tour_dialog_desktop = (active_tab === 0 && !is_mobile) || active_tab === 1;
```

Comment:
@farabi-deriv maybe better to use our constants `DBOT_TABS` instead of numbers from `packages/bot-web-ui/src/constants/bot-contents.ts`?
example:
`active_tab === DBOT_TABS.DASHBOARD`